taipy/core/scenario/_scenario_manager.py

```diff
@@ -270,6 +270,24 @@ def _get_all_by_cycle(cls, cycle: Cycle) -> List[Scenario]:
     def _get_primary_scenarios(cls) -> List[Scenario]:
         return [scenario for scenario in cls._get_all() if scenario.is_primary]
 
+    @classmethod
+    def _sort_scenarios(
+        cls,
+        scenarios: List[Scenario],
+        descending: bool = False,
+        sort_key: Optional[Literal["name", "id", "config_id", "creation_date", "tags"]] = "name",
```

Comment:
```suggestion
        sort_key: Literal["name", "id", "config_id", "creation_date", "tags"] = "name",
```
`sort_key` doesn't accept `None` as a possible value, so it should not has the `Optional` type.